packages/firestore/rollup.config.js

```diff
@@ -84,7 +84,6 @@ const browserPlugins = function () {
         }
       },
       cacheDir: tmp.dirSync(),
-      clean: true,
```

Comment:
I'm glad you figured out the culprit. I thought we need to switch to a different plugin, which means more work.